DeepCat.js

```diff
@@ -34,9 +34,9 @@
 				'deepcat-hintbox-close': 'Zuk&uuml;nftig ausblenden',
 				'deepcat-smallhint-close': 'Ausblenden',
 				'deepcat-hintbox-text': 'Momentane Einschränkung des DeepCat-Gadgets pro Suchbegriff:<br/>' +
-				'Max. Kategoriensuchtiefe: ' + maxDepth + ' / Max. Kategorienanzahl: ' + maxResults + '<br/>' +
+				'Standard Kategoriensuchtiefe: ' + defaultDepth + ' / Max. Kategorienanzahl: ' + maxResults + '<br/>' +
```

Comment:
This is not a very nice translation for "default search depth". "Normale …" would be better, maybe "standardmäßige …".